packages/api/tests/api/index.test.ts

```diff
@@ -9,7 +9,7 @@ jest.mock('../../src/api/settings', () => ({
 describe('/api route registration', () => {
   it('registers a generic catch-all handler', async () => {
     await jest.isolateModulesAsync(async () => {
-      const { api } = require('../../src/api');
+      const { api } = await import('../../src/api');
```

Comment:
Is this the async equivalent to require?

Reply:
They're equivalent but apparently `import` provides better context for the editor